File: 677A/main.cpp
#include<iostream>

using namespace std;

int a[1001];
void solve() {
	int n, h;
	cin >>n >>h;
	for (int i = 0; i < n; i++){
		cin >>a[i];
	}
	int result = 0;
	for (int i = 0; i < n; i++) {
		if (a[i] > h) {
			result += 2;
		} else {
			result += 1;
		}
	}
	cout << result;
	return;
}

int main() {
	solve();
	return 0;
}

File: 677A/main.py
def solve():
    m, n = map(int, input().strip().split())
    ppl_height = list(map(int, input().strip().split()))
    
    result = 0
    for i in range(m):
        if ppl_height[i] > n:
            result += 2
        else:
            result += 1

    print(result)

solve()
